packages/apollo-forest-run/src/ForestRun.ts

```diff
@@ -481,6 +482,8 @@ export class ForestRun extends ApolloCache<any> {
         typeof optimistic === "string",
       );
     }
+    maybeEvictOldData(this.env, this.store);
```

Comment:
Note that eg Relay does it scheduled during some idle time, do you reckon it's not worth here?

Reply:
In my local tests with a limit of 500 operations it takes ~2ms. So 2ms once per 500 cache writes - I decided not to add scheduling related complexity here. We can reconsider later of course.